template-tags.php

```diff
@@ -23,6 +23,7 @@ function referrer_link( $format = '%link', $title = '%title', $sep = '&raquo;',
 }
 
 // Retrieve the category, based on the referrer URL. Useful if you have posts with multiple categories
+// This should be phased out and replaced with get_referrer_term
```

Comment:
You could start that process by making `get_referrer_category()` call `get_referrer_term()` internally.